src/Metrics.Extensions/src/Public/Microsoft.ApplicationInsights.Metrics/MetricSeriesConfigurationForGauge.cs

```diff
@@ -0,0 +1,213 @@
+﻿using System;
```

Comment:
I'm pretty sure stylecop wants `using` be inside `namespace`

Reply:
still separate project